engines/wintermute/base/base_persistence_manager.h

```diff
@@ -45,6 +45,7 @@ class BaseGame;
 class BasePersistenceManager {
 public:
 	char *_savedDescription;
+	TimeDate _savedTimestamp;
```

Comment:
As mentioned elsewhere, please just create a getter.

Reply:
Done in b6acee5.